README.md

```diff
@@ -87,86 +66,98 @@ The following roles need [setting up](https://docs.microsoft.com/en-us/azure/act
+**Input:** Submit a payment request for the customer that is less than the customer's already settled balance.  
+**Output:** The request should now be awaiting debt data enrichment on the web front-end.  
 
-Alternatively the steps can be run manually:
-* run migrations
-  * `docker-compose -f docker-compose.migrate.yaml run --rm database-up`
-* start
-  * `docker-compose up`
-* stop
-  * `docker-compose down` or CTRL-C
+3. **To get a request awaiting ledger assignment or correction.**  
+**Pre-requisite:** Receive the output of 2. above.  
+**Input:** Using the web front-end, add debt enrichment data to the request.  
+**Output:** The request should now be awaiting ledger assignment or correction on the web front-end.  
 
+4. **To get a request awaiting ledger assignment quality check.**  
+**Pre-requisite:** Receive the output of 3. above.  
+**Input:** Using the web front-end sign off on the ledger assignment.  
+**Output:** The request should now be awaiting ledger quality check on the web front-end.  
 
-## Test structure
+5. **To create a new debt dataset.**  
+**Input:** On the web front-end click on the Capture New Dataset button and fill in the details.  
+**Output:** The new dataset should now be added and viewable on the web front-end.  
 
-The tests have been structured into subfolders of `./test` as per the
-[Microservice test approach and repository structure](https://eaflood.atlassian.net/wiki/spaces/FPS/pages/1845396477/Microservice+test+approach+and+repository+structure)
+> Note: 3 and 4 are provisional measures requested by the RPA that may be removed at a future stage.
 
-### Running tests
+>Note that duplicate message detection is based on a `referenceId` property which must be a `UUID`.  If this property is not provided then the `invoiceNumber` property is used instead.
 
-A convenience script is provided to run automated tests in a containerised
-environment. This will rebuild images before running tests via docker-compose,
-using a combination of `docker-compose.yaml` and `docker-compose.test.yaml`.
-The command given to `docker-compose run` may be customised by passing
-arguments to the test script.
+>When authentication is disabled, then the user will automatically be given all roles within the service and assigned a unique user Id.  For testing scenarios where multiple users are required, for example quality checks, then sign in again and a new Id will be assigned.
 
-Examples:
 
+# How to stop the Request Editor
 
-### Run all tests
+The service can be stopped in different ways:
+**Bring the service down**  
 ```
-scripts/test
+docker-compose down
 ```
 
-### Run Acceptance tests
-```shell
-scripts/acceptance arg
+**Bring the service down and clear its data**  
 ```
-- `debug` -- Build docker image and run acceptance test against Browserstack
-- `local` -- Start the service locally, run test on your local machine. Note: You should have chrome browser installed
+docker-compose down -v
+```
```

Comment:
How to stop the Batch Processor
This service can be stopped in different ways:

Bring the service down (link to headers below)
Bring the service down and clear its data
###Bring the service down

docker-compose down
###Bring the service down and clear its data

docker-compose down -v